app/src/androidTest/java/com/github/sdpteam15/polyevents/view/activity/admin/AdminHomeFragmentTest.kt

```diff
@@ -95,4 +96,37 @@ class AdminHubFragmentTest {
         Espresso.onView(ViewMatchers.withId(R.id.btnRedirectUserManagement)).perform(click())
         Intents.intended(IntentMatchers.hasComponent(UserManagementListActivity::class.java.name))
     }
+
+    /*
+    @Test
+    fun itemRequestActivityOpensOnClick() {
+        Espresso.onView(ViewMatchers.withId(R.id.ic_more)).perform(click())
+
+        Espresso.onView(ViewMatchers.withId(R.id.btnRedirectItemReqManagement)).perform(click())
+        Intents.intended(IntentMatchers.hasComponent(ItemRequestActivity::class.java.name))
+    }
+
+    @Test
+    fun itemsAdminActivity() {
+        var availableItems: MutableMap<Item, Int> = mutableMapOf()
+        availableItems[Item(null, "Chocolat", "OTHER")] = 30
+        availableItems[Item(null, "Kiwis", "OTHER")] = 10
+        availableItems[Item(null, "230V Plugs", "PLUG")] = 30
+        availableItems[Item(null, "Fridge (large)", "OTHER")] = 5
+        availableItems[Item(null, "Cord rewinder (15m)", "PLUG")] = 30
+        availableItems[Item(null, "Cord rewinder (50m)", "PLUG")] = 10
+        availableItems[Item(null, "Cord rewinder (25m)", "PLUG")] = 20
+
+        Database.currentDatabase = FakeDatabase
+
+        FakeDatabaseItem.items.clear()
+        for ((item, count) in availableItems) {
+            Database.currentDatabase.itemDatabase!!.createItem(item, count)
+        }
+
+        Espresso.onView(ViewMatchers.withId(R.id.ic_more)).perform(click())
+        Espresso.onView(ViewMatchers.withId(R.id.btn_admin_items_list)).perform(click())
+        Intents.intended(IntentMatchers.hasComponent(ItemsAdminActivity::class.java.name))
+    }
```

Comment:
Do we need to remove these tests?

Reply:
Yes because we will change more fragment to settings so we have to move all button